doc/source/java-jni/README.md

```diff
@@ -153,8 +153,8 @@ The general format is:
 ```bash
 s2i build \
   <git-repo | src-folder> \
-  seldonio/s2i-java-jni-build:0.3.0 \
-  --runtime-image seldonio/s2i-java-jni-runtime:0.3.0 \
+  seldonio/s2i-java-jni-build:0.5.0 \
```

Comment:
Looks good but it seems the images aren't pushed yet

Reply:
I'm currently moving them to the UBI8 base images, as that should have less risk of CVEs. I'll flag the PR as WIP in the meantime.

Reply:
Alright, these images should now be using the UBI8 versions, and should be accessible in Docker Hub :+1:

Reply:
Nice